### PR TITLE
Fix: expose_container_content with correct object relink if substitute collection

### PR DESCRIPTION
**Pull request by Tilix4**
## Brief description
When doing a `expose_container_content`, if a substitute collection is created, object content was not correctly relinked to created collection, was always moved to parent collection.

## Testing notes:
1. Load a simple container which has only objects into its collection
2. Run `expose_container_content`

### CODE REVIEW

File: openpype/hosts/blender/api/ops.py [hunk 1]
```diff
@@ -1069,8 +1069,9 @@ def execute(self, context):
         openpype_containers = context.scene.openpype_containers
         container = openpype_containers.get(self.container_name)
 
-        # In collection, convert to basic
+        # If collection, convert it to regular one
         parent_collection = get_parent_collection(container.outliner_entity)
+        substitute_collection = None
         if isinstance(container.outliner_entity, bpy.types.Collection):
             container.outliner_entity.name += ".old"
             if isinstance(container.outliner_entity, bpy.types.Collection):
```
Comment (kaamaurice) [inline openpype/hosts/blender/api/ops.py]: double `If isinstance(container.outliner_entity, bpy.types.Collection):`

File: openpype/hosts/blender/api/ops.py [hunk 2]
```diff
@@ -1082,10 +1083,13 @@ def execute(self, context):
                     container.outliner_entity.children, substitute_collection
                 )
 
-        # Unlink entity from scene
+        # Move objects to either substituted collection or the parent one
         link_to_collection(
-            container.outliner_entity.objects, parent_collection
+            container.outliner_entity.objects,
```
Comment (kaamaurice) [inline openpype/hosts/blender/api/ops.py]: If `not isinstance(container.outliner_entity, bpy.types.Collection)` container.outliner_entity.objects could raise an AttributeError ?

**Review by kaamaurice — CHANGES_REQUESTED** [review]

File: openpype/hosts/blender/api/ops.py (hunk 2 above)
Comment (Tilix4) [reply openpype/hosts/blender/api/ops.py]: You're right, I've inverted `objects` and `children`.

**Review by kaamaurice — APPROVED** [review]
LGTM

**Pull request merged**